src/schemas/TaxBaseInput.yaml

```diff
@@ -21,5 +21,5 @@ properties:
     example: "French standard VAT"
   applied_to_organization:
     type: boolean
-    description: Set to `true` if the tax is used as one of the organization's default. Also this tax will be applied to the default entity of the organization. This field is deprecated and will be removed in a future version. Please use `billing_entity/:code/manage_taxes` endpoint instead
+    description: Set to `true` if the tax is used as one of the organization's default. Also this tax will be applied to the default entity of the organization. This field is deprecated and will be removed in a future version.
```

Comment:
Thank you!!!
I was thinking if we should specify that they should use `billing_entities/:code/update` instead now to apply taxes on billing entities, what do you think?